yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/launcher/DockerComputerSSHLauncher.java

```diff
@@ -68,6 +81,10 @@ public void appendContainerConfig(DockerSlaveTemplate dockerSlaveTemplate, Creat
 
         createCmd.withPortSpecs(sshPort + "/tcp");
 
+        if (restartSlave) {
+            createCmd.withRestartPolicy(RestartPolicy.alwaysRestart());
```

Comment:
if we imagine that docker create allows to have different restart policies i.e. restart 10 times, then ssh setting with checkbox will be not correct. Logically user should enable restart in docker create settings (just docker setting) and in launcher (slave.jar option). Just trying to keep things separate, also that may happen that user will set restart once, but launcher will be invisibly adding setting... that also may look unexpected.